test/stripe/product_test.rb

```diff
@@ -17,7 +17,9 @@ class ProductTest < Test::Unit::TestCase
 
     should "be creatable" do
       _ = Stripe::Product.create(
```

Comment:
Do you mind assigning the result to `product`, and adding an assert to check that `product` is a `Stripe::Product`? This is not related to your issue but it's what we do ~everywhere in tests, not sure why this is not the case here.